video/rest/compositions/get-composition-media-file/get-composition-media-file.5.x.cs

```diff
@@ -5,21 +5,25 @@
 using System.IO;
 using System.Net;
 using System.Text;
+using Twilio;
 using Newtonsoft.Json;
 
 class Example
 {
     static void Main(string[] args)
     {
-      // Find your API Key SID and Secret at twilio.com/console
-      const string apiKeySid = Environment.GetEnvironmentVariable("TWILIO_API_KEY");
-      const string apiKeySecret = Environment.GetEnvironmentVariable("TWILIO_API_KEY_SECRET");
+      // Find your Account SID and Auth Token at twilio.com/console
+      // and set the environment variables. See http://twil.io/secure
+      string accountSid = Environment.GetEnvironmentVariable("TWILIO_ACCOUNT_SID");
+      string authToken = Environment.GetEnvironmentVariable("TWILIO_AUTH_TOKEN");
+
+      TwilioClient.Init(accountSid, authToken);
 
       const string compositionSid = "CJXXXX";
       const string uri = $"https://video.twilio.com/v1/Compositions/{compositionSid}/Media?Ttl=3600";
 
       var request = (HttpWebRequest)WebRequest.Create(uri);
-      equest.Headers.Add("Authorization", "Basic " + Convert.ToBase64String(Encoding.ASCII.GetBytes(apiKeySid + ":" + apiKeySecret)));
+      request.Headers.Add("Authorization", "Basic " + Convert.ToBase64String(Encoding.ASCII.GetBytes(apiKeySid + ":" + apiKeySecret)));
```

Comment:
After changing the `apiKeySid` and `apiKeySecret` variables here to be the account sid and auth token, this *mostly* works. It doesn't download an mp4, but instead downloads ... I'm not exactly sure what. Maybe the binary data? I'll look more tomorrow :)

Reply:
But, it runs and gets the file from Twilio